stacks/services.yml

```diff
@@ -18,6 +18,7 @@ services:
       - APACHE_DOCUMENTROOT=/var/www/${DOCROOT:-docroot}
       - APACHE_BASIC_AUTH_USER
       - APACHE_BASIC_AUTH_PASS
+      - APACHE_SERVERNAME
```

Comment:
@lmakarov pls review

Reply:
I read the documentation on this https://httpd.apache.org/docs/2.4/vhosts/name-based.html and it doesn't look like something we should allow to customize to me.